src/tests/useQuery-SSR.test.js

```diff
@@ -1,22 +1,20 @@
+/**
+ * @jest-environment node
+ */
+
 import { cleanup } from '@testing-library/react'
 import React from 'react'
 import { renderToString } from 'react-dom/server'
+import { usePaginatedQuery, queryCache } from '../index'
 
 describe('useQuery SSR', () => {
-  beforeEach(() => {
-    const windowSpy = jest.spyOn(global, 'window', 'get')
-    windowSpy.mockImplementation(() => undefined)
-  })
-
   afterEach(() => {
+    queryCache.clear()
     cleanup()
```

Comment:
Is this still required? I thought I saw somewhere that it's not. 🤷‍♂️

Reply:
Indeed, it's not required starting from RTL v9
https://github.com/testing-library/react-testing-library/releases/tag/v9.0.0
Still required, if test suite doesn't have afterEach set though